Project_Templates/ClassesV1/Classes/Model.swift

```diff
@@ -12,23 +12,24 @@ class Model {
 
     // MARK: - Private properties
     
-    private var cdStack: CDStack!
+    fileprivate var cdStack: CDStack!
     
-    lazy private var applicationDocumentsDirectory: NSURL = {
-        
-        return NSFileManager.defaultManager().URLsForDirectory(.DocumentDirectory, inDomains: .UserDomainMask)[0] as NSURL
+    lazy fileprivate var applicationDocumentsDirectory: URL = {
+
+        return FileManager.default.urls(for: .documentDirectory, in: .userDomainMask)[0] as URL
         }()
     
     // MARK: - Public properties
-    
-    lazy var frc_example: NSFetchedResultsController = {
-        
-        // Use this as a template to create other fetched results controllers
-        let frc = self.cdStack.frcForEntityNamed("Example", withPredicateFormat: nil, predicateObject: nil, sortDescriptors: "attribute1,true", andSectionNameKeyPath: nil)
-        
+
+    //Use this as a template to create other fetched results controllers. Replace `Example` with the entity type you are fetching
+    lazy var frc_example: NSFetchedResultsController<NSFetchRequestResult> = {
+        let fetchRequest = NSFetchRequest<NSFetchRequestResult>(entityName: "Example")
```

Comment:
I noticed here I used inferred typing, not sure if that is confusing for students. (vs. let fetchRequest: NSFetchRequest...)